components/homme/src/theta-l/share/prim_advance_mod.F90

```diff
@@ -659,23 +659,6 @@ subroutine advance_hypervis(elem,hvcoord,hybrid,deriv,nt,nets,nete,dt2,eta_ave_w
            stens(:,:,k,3,ie)=-nu  *stens(:,:,k,3,ie) ! w
            stens(:,:,k,4,ie)=-nu_s*stens(:,:,k,4,ie) ! phi
         enddo
-        if (nu_top>0 .and. hypervis_subcycle_tom==0) then
```

Comment:
Was nu_top kept in F90 just for allowing comparison with C++? Meaning, actual runs in F90 were already no longer using nu_top (and relying only on hv_subcycle_tom)?

Reply:
I am not sure what you mean, but this particular code was in F version is to save some mpi rounds by tying them with mpi rounds in general HV. This code is not used, so, I removed it to not to confuse us to test it. I removed similar code from xx version too. 

Overall, we need nu_top  and hv_subcycle_tom in both codes.